client/src/components/InputTextField/InputTextField.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./InputTextField.css";
+
+const InputTextField = props => {
+  let { type, name, placeholder, onChange } = props;
+  let classList = `input-field-container-${type}`;
+  InputTextField.defaultProps = {
+    type: "default"
+  };
+
+  return (
+    <div className={classList}>
+      {type === "search" ? (
+        <svg viewBox="-4 -2 24 24">
+          <path
+            fill="#bababa"
+            d="M9.5,3A6.5,6.5 0 0,1 16,9.5C16,11.11 15.41,12.59 14.44,13.73L14.71,14H15.5L20.5,19L19,20.5L14,15.5V14.71L13.73,14.44C12.59,15.41 11.11,16 9.5,16A6.5,6.5 0 0,1 3,9.5A6.5,6.5 0 0,1 9.5,3M9.5,5C7,5 5,7 5,9.5C5,12 7,14 9.5,14C12,14 14,12 14,9.5C14,7 12,5 9.5,5Z"
+          />
+        </svg>
+      ) : null}
+      <input type={props.input} name={name} placeholder={placeholder} onChange={onChange} />
+    </div>
+  );
+};
```

Comment:
Can we use the `type` property for the `type` attribute on the `input` element and rename `type` (i.e. the type of input component) to something else?

Reply:
Switched everything around per your comment